### PR TITLE
Uniformize app name

### PR DESCRIPTION
**Pull request by Deewarz**
The most important commit ever 😅​

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 <br />
 <div align="center">
-  Mafia: Definitive Edition Multiplayer!
```
Comment (Segfaultd) [inline README.md:12]: Not sure about the branding w'd like to use on the mod

File: code/client/src/core/hooks/render_device.cpp [hunk 2]
```diff
@@ -89,7 +89,7 @@ int64_t C_WindowProcHandler__CreateMainWindow(void *_this, SDK::ue::C_Applicatio
     MafiaMP::Game::gGlobals.window = appWin32->m_pWindow;
 
     // Update the main window title asap
-    SetWindowTextA(MafiaMP::Game::gGlobals.window, "Mafia: Advanced Multiplayer Edition");
```
Comment (Segfaultd) [inline code/client/src/core/hooks/render_device.cpp:92]: This was an internal joke :D Not sure if we should keep it atm

**Review by Segfaultd — COMMENTED** [review]
@zpl-zak any opinion on it?

**Comment by Deewarz** [comment]
> @zpl-zak any opinion on it?

The goal of this PR is just to uniformize the name everywhere.

We will have time to discuss about the final name in the future :)

File: code/client/src/core/hooks/render_device.cpp (hunk 2 above)
Comment (zpl-zak) [reply code/client/src/core/hooks/render_device.cpp:92]: Mafia Super Duper Advanced Multiplayer Definitive Edition

**Comment by zpl-zak** [comment]
> > @zpl-zak any opinion on it?
> 
> The goal of this PR is just to uniformize the name everywhere.
> 
> We will have time to discuss about the final name in the future :)

I'm ok with Mafia: DE Multiplayer, or Mafia Multiplayer.

**Comment by Deewarz** [comment]
> > > @zpl-zak any opinion on it?
> > 
> > 
> > The goal of this PR is just to uniformize the name everywhere.
> > We will have time to discuss about the final name in the future :)
> 
> I'm ok with Mafia: DE Multiplayer, or Mafia Multiplayer.

"Mafia: DE Multiplayer" is more explicit by not pleasant to read...

"Mafia Multiplayer" can you confirm without the `:` after Mafia?

**Comment by zpl-zak** [comment]
Ye w/o colon is fine

**Comment by Deewarz** [comment]
@Segfaultd What is your final word that I make the change?

**Pull request merged**